File: Pharmacy_project/project_BE/project_crud/project_crud/src/main/java/com/project_crud/service/PrescriptionService.java
package com.project_crud.service;

import java.util.List;

import com.project_crud.model.Prescription;

public interface PrescriptionService {

    // Add Prescription
    public Prescription addPrescription(Prescription prescription);

    // View Prescriptions
    public List<Prescription> viewPrescriptions();

    // Delete Prescription
    public String deletePrescription(int id);
}

File: Pharmacy_project/frontend_proj/src/services/prescriptionService.js
import api from './api';

export const addPrescription = (data) => api.post('/addPrescription', data);
export const viewPrescriptions = () => api.get('/viewPrescriptions');
export const deletePrescription = (id) => api.delete(`/deletePrescription/${id}`);
export const getPrescriptionsByDoctor = (doctorId) => api.get(`/viewPrescriptions/doctor/${doctorId}`);
